exchanges/bithumb/bithumb_test.go

```diff
@@ -232,11 +232,17 @@ func TestUpdateTicker(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	cp = currency.NewPair(currency.DASH, currency.KRW)
+	cp = currency.NewPair(currency.BTC, currency.KRW)
 	_, err = b.UpdateTicker(cp, asset.Spot)
 	if err != nil {
 		t.Fatal(err)
 	}
+
+	cp = currency.NewPair(currency.DASH, currency.KRW)
+	_, err = b.UpdateTicker(cp, asset.Spot)
+	if err == nil {
+		t.Error("expecting DASH to return an error")
+	}
```

Comment:
I don't think we need this in the event they add it back. If you're wanting to test for errors, perhaps consider `DOGE ALICE`

Reply:
good point!

Reply:
In that case I've chosen to remove the error check since they could add any pair 🤷‍♂️